### PR TITLE
add stack overflow URL, edit slack info requested

### PR DESCRIPTION
**Pull request by mmcelaney**
Task #25

### CODE REVIEW

**Comment by mmcelaney** [comment]
@lornajane can you take a look at this when you get a chance?

File: public/home.html [hunk 1]
```diff
@@ -324,23 +324,17 @@
...
-        <div class="form-group">
-          <label for="soid" class="col-sm-3 control-label">Stack Overflow ID</label>
+          <p class="help-block col-sm-9 col-sm-offset-3">We need to know your Stack Overflow identity. Navigate to the Activity Tab in your Stack Overflow profile and enter that URL here so we can tell whether you have answered a question.
+          </p>
+          <label for="soid" class="col-sm-3 control-label">Slack Organization</label>
           <div class="col-sm-9">
             <input v-model="profile.so_id" type="string" class="form-control" id="soid">
           </div>
-          <p class="help-block col-sm-9 col-sm-offset-3">We need to know your Stack Overflow identity too. Enter the number that appears in your Stack Overflow profile URL so we can tell whether you have answered a question.
+          <p class="help-block col-sm-9 col-sm-offset-3">Tell us know which Slack organization you belong to. For example: "ibm-cloudplatform"
```
Comment (lornajane) [inline public/home.html]: "Tell us know" isn't very clear.  Let's change this to "Let us know" or "Tell us" ?

File: public/home.html [hunk 2]
```diff
@@ -324,23 +324,17 @@
...
+            <input v-model="profile.so_id" type="string" class="form-control" id="soid">
           </div>
-        </div>
-        <div class="form-group">
-          <label for="soid" class="col-sm-3 control-label">Stack Overflow ID</label>
+          <p class="help-block col-sm-9 col-sm-offset-3">We need to know your Stack Overflow identity. Navigate to the Activity Tab in your Stack Overflow profile and enter that URL here so we can tell whether you have answered a question.
+          </p>
+          <label for="soid" class="col-sm-3 control-label">Slack Organization</label>
           <div class="col-sm-9">
             <input v-model="profile.so_id" type="string" class="form-control" id="soid">
```
Comment (lornajane) [inline public/home.html]: This is displaying my stackoverflow ID, but should be empty and then store my slack team

Comment (lornajane) [reply public/home.html]: Ha, actually when I try to edit one of these form fields, the other one changes too!  Definitely something not right here :)

**Comment by lornajane** [comment]
I've added a couple of comments, and I also have a question about the slack team.  Is there anywhere we currently use this data or is it for notification or other purposes?  Maybe a question for @ptitzler as he originally suggested the change

**Comment by lornajane** [comment]
Also I see that there are conflicts, you can either rebase your branch or I can try to handle that when I merge, either is OK by me.  Apparently we wildly reformatted a file!

**Pull request closed without merging**